### ISSUE TITLE
Echidna should occasionally intentionally use small values

### ISSUE THREAD

**Issue by agroce** [crytic/echidna#206]
Even with coverage, so long as you don't manually add a memory, this is hard to detect:

```
contract NeedMemory {
  uint256 [] values;

  /* The code commented out below implements an in-contract memory of past values.
     It makes it easy to find the "bug".  Coverage will do the same, by the route of
     memorizing the first random "coverage-increasing" transaction and re-using it.
     Without coverage, however, or this code, the chance of detecting the issue is
     essentially 0, without intentional re-use of generated values.

     It's unclear to what extend coverage can make up for lack of memory in all cases. */

  /*

  uint256 vmem;
  bool everSet = false;

  function set_mem(uint256 v) public {
    everSet = true;
    vmem = v;
  }

  function push_mem() public {
    if (everSet) {
      values.push(vmem);
    }
  }
  */

  function push_it(uint256 v) public {
    values.push(v);
  }

  function echidna_nodups() public returns (bool) {
    uint256 length = values.length;
    uint256 i;
    for (i = 0; i < length; i++) {
      if ((i + 1) < length) {
        if ((values[i] > 2) && ((values[i] * values[i]) < 1048576)) {
          if (values[i] == values[i+1]) {
            return false;
          }
        }
      }
    }
    return true;
  }
}
```

**Comment by agroce** [comment]
Actually, even WITH the manual memory, this is almost impossible to detect!

**Comment by agroce** [comment]
Fixed in #213
